packages/gafl-webapp-service/src/pages/contact/contact/contact.njk

```diff
@@ -139,4 +152,35 @@
       ]
     }) }}
 
+{% endblock %}
+
+{% block bodyEnd %}
+    <script nonce={{nonce}} src="/public/javascript/all-min.js"></script>
+    <script nonce={{nonce}}>window.GOVUKFrontend.initAll()</script>
+    <script nonce={{nonce}}>(function () {
```

Comment:
Can we not do this without using JavaScript?
I put together a quick example of how this can be achieved using CSS only here: https://www.w3schools.com/code/tryit.asp?filename=GFT0GFEY1GR7